web-components/src/components/table/ActionsTable.tsx

```diff
@@ -40,15 +42,22 @@ const BorderLeft = styled('div')(({ theme }) => ({
  *  */
 export type RenderActionButtonsFunc = (i: number) => React.ReactNode;
 
+export type OnActionTableChangeParams = {
+  page: number;
+  rowsPerPage: number;
+  offset: number;
+};
+
```

Comment:
how about `TablePaginationParams` or even `PaginationParams` B instead?

Reply:
Sounds good, will change to `TablePaginationParams ` since we might have other kinds of pagination.